packages/web-components/src/components/va-breadcrumbs/va-breadcrumbs.tsx

```diff
@@ -109,12 +109,17 @@ export class VaBreadcrumbs {
    * @private
    */
   private updateBreadCrumbList(breadcrumbList: Breadcrumb[]) {
```

Comment:
This method was mutating the original breadcrumbsList prop when setting the label. These changes make it so that does not happen and that you can toggle the prop in storybook